.travis.yml

```diff
@@ -20,6 +20,10 @@ script:
 - cargo build --verbose
 - cargo test --verbose
 - cargo doc --verbose
+- cargo build --verbose --target aarch64-apple-ios
+- cargo test --verbose --target aarch64-apple-ios
```

Comment:
`cargo test` won't work with different architectures. If you'd really want to run the tests in an iOS simulator, [`dinghy`](https://github.com/sonos/dinghy) is your friend but you'd still only be able to have CI run it with the `x86_64-apple-ios` target.

Reply:
I've revised the changes, now just doing a build without the test. How do I actually kick off a travis build to test it?